src/nim_status_client.nim

```diff
@@ -55,9 +55,9 @@ proc prepareLogging() =
         except:
           logLoggingFailure(cstring(msg), getCurrentException())
 
-  let defaultLogLvl = if defined(production): LogLevel.INFO else: LogLevel.DEBUG
+  let defaultLogLvl = if defined(production): chronicles.LogLevel.INFO else: chronicles.LogLevel.DEBUG
```

Comment:
1. This is the second definition of default `LOG_LEVEL` value. The first one was here:
https://github.com/status-im/status-desktop/blob/d81bec9faa6c71481f78b5df971085fa89431ab8/src/env_cli_vars.nim#L160
Would be nice to have single place with definition of this, to prevent any confusion.

2. Also, maybe we could `parseEnum` somewhere in `main_constants` as well?

3. I see this in you screenshot:

<img width="640" alt="image" src="https://github.com/status-im/status-desktop/assets/25482501/5be522d9-5c04-4fab-a605-106c7c8a4c50">

```
-log-level     Sets log level to [= defined(production) :$LogLevel.Info else: $LogLevel.Debug].
```

Shouldn't `[...]` be evaluated at compile time? From here the default value is not obvious. And might even be invalid.

Reply:
> probably it makes sense to rename the PR 🙂

The PR name updated.

> 1. This is the second definition of default LOG_LEVEL value. The first one was here:

Updated.

> 2. Also, maybe we could parseEnum somewhere in main_constants as well?

I was thinking about that but decided to do that via new PR later, cause that will bring many new changes that are unrelated to this refactor and the initial issue this PR is about.

> 3. I see this in you screenshot: ... Shouldn't [...] be evaluated at compile time? From here the default value is not obvious. And might even be invalid.

The thing here is that's the way `nim-confutils` works and displays the default value (I was surprised as well, but changing that requires either a change on the `nim-confutils` side or avoiding it and developing the same on the desktop app side, that is not worth). Anyway since I set this `defaultValue: DEFAULT_LOG_LEVEL` the description itself will be different, but ofc the value won't be evaluated.

Reply:
Thanks for the changes!

> Anyway since I set this defaultValue: DEFAULT_LOG_LEVEL the description itself will be different, but ofc the value won't be evaluated.

Maybe manually write a string description there? E.g. 
```
`INFO` in production build, else `DEBUG`
```

Reply:
This part `[=DEFAULT_LOG_LEVEL]` is added by `nim-confutils`, we cannot change it.
This is the part added by us `Can be one of: "ERROR", "WARN", "INFO", "DEBUG", "TRACE".`
I updated the description and added additional info that you proposed.